migrator/models.py

```diff
@@ -85,3 +85,54 @@ class CdnCertificate(CdnBase):
     # certificate is the actual body of the certificate chain
     certificate = sa.Column(postgresql.BYTEA)
     expires = sa.Column(postgresql.TIMESTAMP)
+
+
+class CustomDomainRoute(CustomDomainBase):
+    __tablename__ = "routes"
+
+    guid = sa.Column(sa.Text, primary_key=True)
+    state = sa.Column(sa.Text)
+    domains = sa.Column(sa.Text)
+    challenge_json = sa.column(postgresql.BYTEA)
```

Comment:
Is there a difference between `sa.Column` and `sa.column`?

Reply:
yeah,`sa.column` is a typo